src/CarbonSDK.ts

```diff
@@ -362,6 +363,16 @@ class CarbonSDK {
     return sdk.connectWithMetamask(metamask, walletOpts, metamaskWalletOpts);
   }
 
+  public static async instanceWithRainbowKit(
+    rainbowKit: RainbowKitAccount,
+    sdkOpts: CarbonSDKInitOpts = DEFAULT_SDK_INIT_OPTS,
+    rainbowKitWalletOpts: RainbowKitWalletOpts,
+    walletOpts?: CarbonWalletGenericOpts,
+  ) {
+    const sdk = await CarbonSDK.instance(sdkOpts);
+    return sdk.connectWithRainbowKit(rainbowKit, rainbowKitWalletOpts, walletOpts);
```

Comment:
Since this function is specifically for RainbowKit wallets, think you can declare `isRainbowKit: true` in here directly.

i.e.
```typescript
return sdk.connectWithRainbowKit(rainbowKit, rainbowKitWalletOpts, {
  ...walletOpts,
  isRainbowKit: true,
});
```

Reply:
yes, but I will add it into `connectWithRainbowKit`, not here, thanks